src/run-script.js

```diff
@@ -47,7 +47,7 @@ function getEnv() {
         return envCopy
       },
       {
-        [`SCRIPTS_${script.toUpperCase()}`]: true,
+        [`SCRIPTS_${script.toUpperCase().replace(/-/g, '_')}`]: true,
```

Comment:
If previous `SCRIPTS_` environment variables were considered as public API it wouldn't harm to also include them here - in addition to the new ones.

Reply:
It seems like it never really worked in the first place so I don't think they could have been considered public either way 🙃